src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -70,6 +70,7 @@
         + " -j,--javadocTree            Print Parse tree of the Javadoc comment%n"
         + " -J,--treeWithJavadoc        Print full Abstract Syntax Tree of the file%n"
         + " -o <arg>                    Sets the output file. Defaults to stdout%n"
+        + " -omitIgnoredModules <arg>   Sets ability to not run modules that are ignored.%n"
```

Comment:
@rnveach 
> If we want to change this to no argument, what should it's name be? We are omitting modules by default.

1. executeIgnoredModules, short: eim
2. runIgnoredModules, short: rim
3. processIgnoredModules, short: pim

In general boolean options should not have arguments in CLI.